src/NzbDrone.Core/Tv/AddSeriesOptions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+
+namespace NzbDrone.Core.Tv
+{
+    public class AddSeriesOptions
+    {
+        public bool SearchForMissingEpisodes { get; set; }
+        public bool IgnoreEpisodesWithFiles { get; set; }
+        public bool IgnoreEpisodesWithoutFiles { get; set; }
+        public List<int> IgnoreSeasons { get; set; }
+
+        public AddSeriesOptions()
```

Comment:
Aren't we missing 'IgnoreAlreadyAired' or is that IgnoreEpisodesWithFiles+IgnoreEpisodesWithoutFiles?